index.html

```diff
@@ -20,10 +20,306 @@
+
+      <!--Footer-->
+      <footer>
+        <h2 class="sr-only">Site Footer</h2>
+          <div class="footer">
+            <div class="footer--content">
+              <!--Social Media Icons-->
+              <div class="socials">
+                <ul>
+                  <li>
+                    <img
+                    class="icons"
+                    src="./images/Instagram.png"
+                    alt="Instagram Icon">
+                  </li>
+                  <li>
+                    <img
+                    class="icons"
+                    src="./images/twitter-logo-black-and-white.png"
+                    alt="Twitter Icon">
+                  </li>
+                  <li>
+                    <img
+                    class="icons"
+                    src="./images/YouTube.png"
+                    alt="YouTube Icon">
+                  </li>
+                  <li>
+                    <img
+                    class="icons"
+                    src="./images/Pinterest.png"
+                    alt="Pinterest Icon">
+                  </li>
+                  <li>
+                    <img
+                    class="icons"
+                    src="./images/Facebook.png"
+                    alt="Pinterest Icon">
+                  </li>
+                </ul>
+              </div>
+              <!--Footer Navigation Lists-->
+              <div class="footer--navigation">
+                <div class="footer--nav--titles">
+                  <ul>
+                    <li class="nav--title">News</li>
+                    <li class="nav--content">Season 2</li>
+                    <li class="nav--content">Latest</li>
+                    <li class="nav--content">Articles</li>
+                  </ul>
+                  <ul>
+                    <li class="nav--title">Watch</li>
+                    <li class="nav--content">Sign in to Netflix</li>
+                    <li class="nav--content">Sign up for Netflix</li>
+                    <li class="nav--content">Stream with others</li>
+                  </ul>
+                  <ul>
+                    <li class="nav--title">Shop</li>
+                    <li class="nav--content">Clothes</li>
+                    <li class="nav--content">Accessories</li>
+                    <li class="nav--content">Collectibles</li> 
+                  </ul>  
+                  <ul>
+                    <li class="nav--title">Quizzes and Features</li>
+                    <li class="nav--content">Quizzes</li>
+                    <li class="nav--content">Chat</li>  
+                    <li class="nav--content">Leave a review</li>  
+                  </ul>
+                  <ul>
+                    <li class="nav--title">Help</li>
+                    <li class="nav--content">FAQs</li>
+                    <li class="nav--content">Contact us</li>  
+                    <li class="nav--content">Report</li>
+                  </ul>                  
```

Comment:
To improve this code you could wrap the whole list inside a nav element since it's all site wide navigation. 

```html 
<nav>
  <ul>
    <li>...</li>
    <li>...</li>
    ... 
  </ul>
</nav>